frontend/app/components/TaskDescription.server.vue

```diff
@@ -0,0 +1,38 @@
+<script setup lang="ts">
+import { FetchError } from 'ofetch'
+
+const description = useState('task-description', () => '')
+
+const route = useRoute()
+const task_id = String(route.params.id ?? 'simple-task-example')
```

Comment:
It is weird behaviour (you are to js), just return error, run request for "" or "undefined", not just random name